### DISCUSSION TITLE
Downloaded attachments stored in Folders based on Email Subject

### DISCUSSION THREAD

**Discussion by felixlu07** [munir131/attachment-downloader#27] [Ideas]
Anyway to save the attachments downloaded under a folder, with the Email Subject as the Folder name??

**Comment by munir131** [comment]
@felixlu07 Not now. But, you can store based on `from` emails. or you can make label for subject and use label for use as directory name. If you have little knowlege of JS then you can add subject name here - https://github.com/munir131/attachment-downloader/blob/master/index.js#L168
